src/dotnet/HQ.Abstractions/Staff/UpsertStaffV1.cs

```diff
@@ -23,11 +23,13 @@ public class Request
         public DateOnly? EndDate { get; set; }
         public string? FirstName { get; set; }
         public string? LastName { get; set; }
-        public string? Email { get; set; }
+        public bool? CreateUser { get; set; }
```

Comment:
This doesn't need to be nullable, in this case it's either checked or not. The primary use case for nullable booleans is on our Get requests where we want to query for all, where true, or where false (3 options). But for a user create/edit form, a non-nullable boolean is appropriate.